plugin/trino-phoenix5/src/test/java/io/trino/plugin/phoenix5/PhoenixQueryRunner.java

```diff
@@ -133,11 +134,13 @@ private static Session createSession()
     public static void main(String[] args)
             throws Exception
     {
+        SharedResource.Lease<TestingPhoenixServer> instance = TestingPhoenixServer.getInstance();
```

Comment:
I think you should use `try-with-resources` and instance should be closed just before ending `main` method

Reply:
You are right